lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -711,18 +711,23 @@ function dosomething_reportback_save($values, $user = NULL) {
   }
   else {
     $entity = entity_load_single('reportback', $values['rbid']);
+    $reportback_item = entity_load_single('reportback_item', $values['fid']);
   }
 
   try {
     // Set entity properties.
     dosomething_reportback_set_properties($entity, $values);
 
-    // Set entity files.
-    dosomething_reportback_set_files($entity, $values);
-
+    if ($reportback_item) {
```

Comment:
Since the assignment of `$reportback_item` happens in an `if/else` statement it may meant that it doesn't get created and assigned, so when you check for it here there'll be an error if it doesn't exist. Best to wrap this check in `isset($reportback_item)`!